src/Application/inc/Calculator.h

```diff
@@ -22,17 +25,25 @@ class Calculator
     public:
         /**
          * @brief initiates calculator and works with the parser to return a result object
+         * @param [in] history a pointer to a created history class usd to track completed expressions
         */
-        void runCalculator();
+        void runCalculator(std::shared_ptr<calculator::History> &history);
 
         /**
          * @brief Runs a switch statement with entered operator to calculate 2 numbers
          * @param [in] parsedExpression an expression object containing 2 numbers and an operators
          * @return the calculated float answer
         */
         float calculate(const Expression &parsedExpression);
+
+        /**
+         * @brief checks user input to see if they want to calculate again, quit, or view history
+         * @param [in] history a pointer to a created history class usd to track completed expressions
+         * @return bool for whether or not calculator should keep running
+        */
+        bool restart(std::shared_ptr<calculator::History> &history);
 };
 
-}//namespace calculator
+} //namespace calculator
 
-#endif  // CALCULATOR_H
+#endif //CALCULATOR_H
```

Comment:
Im not positive if its just github but make sure you have the extra line at the end of the file. I won't comment on every one cause I'm pretty its just github presenting them wrong.